src/storage.js

```diff
@@ -0,0 +1,66 @@
+import {getCookie, setCookie} from "./cookies.js";
+
+/**
+ *
+ * @param useCookies - A String - 'true' for the top level domain, otherwise pass in the subdomain. Leave blank to use
+ * local storage
+ * @param allowPersistence - true is the most common. This generally false when a user has not accepted cookie tracking
+ * @constructor
```

Comment:
You can include the type before the param name. If you wrap the param name with brackets `[]`, the param becomes optional and you can also provide a default value. Example `@param {boolean} [allowPersistence=true]`
```suggestion
 * @param {string|undefined} useCookies - 'true' for the top level domain, otherwise pass in the subdomain. Leave blank to use
 * local storage
 * @param {boolean} allowPersistence - true is the most common. This generally false when a user has not accepted cookie tracking
 * @constructor
```

Reply:
Added to new pr